node_packages/bazed-svelte/.eslintrc.cjs

```diff
@@ -17,4 +17,10 @@ module.exports = {
     es2017: true,
     node: true,
   },
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-inferrable-types": "off",
+    "no-unused-vars": "off",
```

Comment:
Similarily, maybe make this a warn?

Reply:
Eh, warnings _should_ be errors in CI, so that'd then mean that they're still errors 🤷🏻‍♂️ At some point we might wanna get rid of these errors, but until we're _very_ clear on our data model, the odd `as any` in a few places is convenient. If you think we actually wanna get rid of all of those, and get rid of all you `let foo: number` type of things, then sure -- but I doubt that's a good idea

Reply:
And no-unused-vars needs to be off, as it otherwise conflicts with eslint-typescripts no-unused-vars check, which properly accounts for variables in type definitions and stuff